src/sagemaker/amazon/validation.py

```diff
@@ -45,4 +49,5 @@ def validate(value):
 
 isint = istype(int)
 isbool = istype(bool)
+isstr = istype(str)
```

Comment:
These validations are gone. The pull request itself is stating there are conflicts, can you please update your branch to fix the conflicts?

Refer to this PR: https://github.com/aws/sagemaker-python-sdk/commit/54b3830dd9aa5da789c96d813386e9dbb8f5c810#diff-0bb270a0ed6827421dc5669020eb6427  to check what the changes to the hyperparameters are.